apps/web/src/server/app/lpa/lpa.pipes.js

```diff
@@ -0,0 +1,69 @@
+import { body } from 'express-validator';
+
```

Comment:
Shouldn't the validators in this file also be calling .trim() and .escape() to sanitize their input?

Reply:
Possibly, although I'd say it would be better to sanitize the values themselves before they are checked by the validator. This would be a good point to discuss with Radu when he's back as it's his setup, I was just trying to keep things consistent with the pre-existing code in the validation journey pipes file.

Reply:
Yeah, this is what I mean. The inputs should be sanitized/trimmed first, else notEmpty could pass for a string with an empty space, I guess. e.g.

```
body('someprop')
  .escape()
  .trim()
  .notEmpty()
  ...
```
I see there's not really any escaping/trimming being done, so you can leave it as we'll have to address the rest of the validators anyway.

Reply:
Tagging @radum for visibility. Maybe we should add a ticket to cover updating the validation pipes for all journeys at once?

Reply:
Updated all relevant pipes in the validation and case officer journeys with this. Let me know if you spot any I've missed.